modules/nf-core/mmseqs/createindex/main.nf

```diff
@@ -20,10 +19,11 @@ process MMSEQS_CREATEINDEX {
 
     script:
     def args = task.ext.args ?: ''
-    def db_path_name = db_name ? "${db}/${db_name}": "${db}/${db}"
     """
+    DB_NAME=`basename \$(find -L ./$db/ -name "*_seq.tsv") | sed 's/_seq\\.tsv\$//'`
```

Comment:
A suggestion 1) so that you can still use `$db_path_name` in the `mmseqs createindex` command below, 2) in case `$db` is an absolute path, and 3) in case `$db` has whitespace or funky characters.
```suggestion
    db_path_name=\$(find -L "$db/" -name "*_seq.tsv" | sed 's/_seq\\.tsv\$//')
```